mkinitcpio

```diff
@@ -277,7 +280,7 @@ build_uefi(){
                 break
             fi
         done
-    elif [[ ! -f "$uefisub" ]]; then
+    elif [[ ! -f "$uefistub" ]]; then
```

Comment:
This is a separate bugfix and should not be hidden in this commit :)

Reply:
Sure, moved it to #80. Was broken in some other places anyway.

Reply:
Thanks! You didn't drop the commit to the master branch though :)

Reply:
Should disappear from this PR once #80 gets merged I think. At least it's the same commit in both PRs. If not, I'll fix that. At the moment I get the feeling, that this PR won't get through anyway ;-)

Reply:
Please don't let my opinion discourage you :) @grazzolini is still the maintainer and if he want's this I'll help ensure it's as correct as it can be despite my objections.